### DISCUSSION TITLE
Run Miner

### DISCUSSION THREAD

**Discussion by 16871560** [filecoin-project/lotus#9568] [Build and Sync]
ERROR: creating node: starting node here: could not build arguments for function "github.com/filecoin-project/lotus/node/modules".HandleDeals (/Users/benyoung/Desktop/filecoin/node/modules/storageminer.go:360): failed to build storagemarket.StorageProvider: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *storedask.StoredAsk: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build storagemarket.StorageProviderNode: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *sectorblocks.SectorBlocks: missing dependencies for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): missing type: sectorblocks.SectorBuilder (did you mean *sectorblocks.SectorBlocks?)

**Comment by Reiers** [comment]
Hi @16871560 (Ben)
Can you share your build cmd, and Go version?

**Reply by 16871560** [reply]
go version go1.18.5 darwin/amd64     cd /filecoin/cmd/lotus-miner && go build  -o lotus-market ./  Tkx

**Comment by 16871560** [comment]
I  had commented code  func paramfetch () (cuz I just enabled markets)  and build  go build -o lotus-market , it seems ok, after then,  I export the fullnode api token, and run lotus-market  init  --actor t04xxxx  and I can get the message of ChangePeerID， everything is fine. finally  I  run lotus-market run . it  occurs the error  as I have posted
